source/common/config/datasource.h

```diff
@@ -95,6 +99,10 @@ class RemoteAsyncDataProvider : public Config::DataFetcher::RemoteDataFetcherCal
   AsyncDataSourceCb callback_;
   const Config::DataFetcher::RemoteDataFetcherPtr fetcher_;
   Init::TargetImpl init_target_;
+
+  Event::TimerPtr retry_timer_;
+  BackOffStrategyPtr backoff_strategy_;
+  uint32_t retries_remaining_{1};
```

Comment:
If you initialize this in the constructor list, you can remove `{1}` here and remove the future possibility that default RetryCount != 1.

Reply:
Done.